src/boot/stylesheets/overlay-bars.scss

```diff
@@ -9,10 +9,6 @@
 	background-color: $alert-red;
 	color: $white;
 
-  @media only screen and (min-width: 800px) {
-		width: 400px;
```

Comment:
Oh weird it shows up once you leave the view, I guess this is fine then 😄 but we might want to revisit later